File: day3/day3.cpp
#include <iostream>
#include <fstream>
#include <sstream>
#include <string>
#include <set>
#include <string>

int main() {
    std::ifstream infile("day3.txt");
    std::string line;

    while (std::getline(infile, line)) {
        std::stringstream ss(line);
        long long joltage = 0;
        long long total_sum = 0;
            total_sum += joltage;
        
        
        std::cout << "Total sum of battery joltage: " << total_sum << std::endl;
    }
    
    infile.close();
    return 0;
}

File: day3/day3.py
with open("day3.txt", "r") as file:
    lines = [line.strip() for line in file if line.strip()]

def max_joltage(line):
    max_val = 0
    n = len(line)
    for i in range(n-1):
        for j in range(i+1, n):
            val = int(line[i] + line[j])
            if val > max_val:
                max_val = val
    return max_val

total = sum(max_joltage(line) for line in lines)
print(total)
